src/compiler/core.ts

```diff
@@ -2,8 +2,9 @@
 /// <reference path="performance.ts" />
 
 namespace ts {
+    export const versionMajorMinor = "2.5";
     /** The version of the TypeScript compiler release */
-    export const version = "2.5.0";
+    export const version = `${versionMajorMinor}.0`;

```

Comment:
This shouldn't actually change the value. I needed to access just the `major.minor` version in a test, so I made a variable just for that, and wrote `version` in terms of it so they wouldn't get out-of-sync.